src/components/Counter.jsx

```diff
@@ -0,0 +1,18 @@
+export const maxChars = 120;
+
+const Counter = ({ chars, onChange }) => {
+  const charsLeft = maxChars - chars.length;
+
+  return (
+    <>
```

Comment:
This fragment JSX is not needed because it only has one child, the p tag. It's only useful if there will be more than one child component at the same level.